### PR TITLE
Add a settings menu to the FSLeyes plugin

### LINKED ISSUE #458: Add a settings menu to the plugin
PR #462

I propose adding a settings menu to the FSLeyes GUI.
In this menu, the user could:
- Change some values that were previously hardcoded (exemple: https://github.com/neuropoly/axondeepseg/issues/451)
- Change the suffix we look for when loading a mask. Currently, we only support loading masks that contain both axon and myelin in the image. A given suffix would determine if a certain image contains only axon or myelin.
- Determine if we use the ellipse or circle for computing morphometrics
- Check the pixel size of the selected image

Feel free to add any other suggestions

### PR DESCRIPTION
**Pull request by Stoyan-I-A**
Still a work in progress.
Fixes #458 

See #458 for a screenshot

### CODE REVIEW

**Comment by Stoyan-I-A** [comment]
This PR is ready for review. It only contains the settings required for the short term.
The code does need refactoring and more settings could be added, but those will be addressed in a future issue.

File: ads_plugin.py [hunk 1]
```diff
@@ -640,6 +655,103 @@ def on_compute_morphometrics_button(self, event):
...
+        done_button.Bind(wx.EVT_BUTTON, self.on_done_button)
+        sizer_done_button.Add(done_button, flag=wx.SHAPED, proportion=1)
+        frame_sizer_h.Add(sizer_done_button)
+
+        self.settings_frame.SetSizer(frame_sizer_h)
+        self.settings_frame.Show()
+
+    def on_overlap_value_changed(self, event):
+        self.overlap_value = self.overlap_value_spinCtrl.GetValue()
+        print(self.overlap_value)
```
Comment (mariehbourget) [inline ads_plugin.py:733]: Temporary print to remove?

**Review by mariehbourget — DISMISSED** [review]
I tested the new settings menu, everything is working as expected.
Minor comments about temporary prints to remove.
Thanks @Stoyan-I-A!

**Review by mathieuboudreau — APPROVED** [review]
Just reconfirming @mariehbourget 's approval. Only changes I made were merge from master (resolved conflict in version number), and removed the temporary print statements mentioned by MH.

**Pull request merged**